framework/src/transfers/Transfer.C

```diff
@@ -67,7 +67,7 @@ Transfer::find_sys(EquationSystems & es, const std::string & var_name)
     if (es.get_system(i).has_variable(var_name))
       return &es.get_system(i);
 
-  mooseError("Unable to find variable " + var_name + " in any system.");
+  ::mooseError("Unable to find variable " + var_name + " in any system.");
```

Comment:
Inside a static member function, the compiler gets confused about which (error) function to call (i.e. member function vs global function).